### PR TITLE
fix: delete gigachad

### PR DESCRIPTION
**Pull request by sussycatgirl**
## Please make sure to check the following tasks before opening and submitting a PR

* [ ] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
* [ ] I have tested my changes locally and they are working as intended
* [ ] These changes do not have any notable side effects on other Revolt projects
* [ ] (optional) I have opened a pull request on [the translation repository](https://github.com/revoltchat/translations)
* [ ] I have included screenshots to demonstrate my changes

### CODE REVIEW

**Review by infi — APPROVED** [review]
Does whats intended with the PR.

**Review by Rexogamer — APPROVED** [review]
LGTM

**Comment by SingleSlice** [comment]
Looks good to me

**Comment by amycatgirl** [comment]
looks good to me as well

**Comment by tulpenkiste** [comment]
LGTM :trollface:

**Comment by lo-kiss** [comment]
LGTM

**Comment by NyaomiDEV** [comment]
lgtm

File: src/assets/emojis.ts [hunk 1]
```diff
@@ -1941,8 +1941,6 @@ export const emojiDictionary = {
         charle: "custom:charle.gif",
         sadge: "custom:sadge.webp",
         sus: "custom:sus.webp",
-        chade: "custom:chade.gif",
-        gigachad: "custom:gigachad.webp",
```
Comment (insertish) [inline src/assets/emojis.ts:1945]: Remove these two changes and it'll be ready to merge.

**Review by insertish — CHANGES_REQUESTED** [review]

File: src/assets/emojis.ts (hunk 1 above)
Comment (insertish) [reply src/assets/emojis.ts:1945]: Other than that LGTM

**Review by drendog — APPROVED** [review]

**Comment by Zomatree** [comment]
🇹🇩

**Comment by searinminecraft** [comment]
eh... looks good to me

**Comment by Rexogamer** [comment]
> 

gigaromania

**Comment by Rexogamer** [comment]
> giga🇹🇩

gigaromania

**Comment by insertish** [comment]
Fixed in 9eca58dda18e6f7fe401d4b9cdd482d3dce3d788

**Comment by amycatgirl** [comment]
holy fuck he did it

**Pull request closed without merging**